students/alex_skrn/lesson04/mailroom.py

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env python3
+
+"""Mailroom - Part 2 - Dicts, Files."""
+import os
+import datetime
+import tkinter as tk
+from tkinter import filedialog
+
+donors = {'Aristarkh Lentulov': [4.5, 5.0],
+          'El Lissitzky': [34.2, 30.0, 35.5],
+          'Kazimir Malevich': [15.0, 20.25, 12.25],
+          'Marc Chagall': [148.75, 155.0],
+          'Wassily Kandinsky': [75.0, 50.5, 60.4],
+          }
+
+
+# DONOR-RELATED FUNCTIONS
+def add_donation(name, amount):
+    """Add a donation for a named donor."""
+    if name in donors:
+        donors[name].append(float(amount))
+    else:
+        donors[name] = [float(amount)]
```

Comment:
You probably want to wrap these in a `try except` for `ValueError` if the person doesn't enter a number